MockPSConsole/MockPSConsole.csproj

```diff
@@ -17,8 +17,8 @@
     <PackageReference Include="PowerShellStandard.Library" version="5.1.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
-    <PackageReference Include="Microsoft.PowerShell.SDK" version="7.1.0" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.PowerShell.SDK" version="7.2.0-preview.3" />
```

Comment:
What error does the user get if they try to use plugin with PS < 7.2?

Reply:
They will get the following error:
```
PS:4> Set-PSReadLineOption -PredictionSource HistoryAndPlugin
Set-PSReadLineOption: The prediction plugin source is not supported in this version of PowerShell. The 7.2 or a higher version of PowerShell is required to use this source.
```